build.sbt

```diff
@@ -555,6 +555,7 @@ lazy val publishableSettings = Def.settings(
   publishArtifact.in(Compile) := true,
   publishArtifact.in(Test) := false,
   publishMavenStyle := true,
+  bintrayReleaseOnPublish := !isCustomRepository,
```

Comment:
Yes, this works for us at Twitter. 

I'm getting a new warning saying `[warn] You must run bintrayRelease once all artifacts are staged`, but I can live with it.